src/main/java/seedu/address/model/reservation/Pax.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.reservation;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Reservation's pax in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPax(String)}
+ */
+public class Pax {
+
+    public static final String MESSAGE_CONSTRAINTS = "Pax should only be a positive number";
+    public static final String VALIDATION_REGEX = "\\d+";
+    public final String value;
+
+    /**
+     * Constructs a {@code Pax}.
+     *
+     * @param pax A valid pax.
+     */
+    public Pax(String pax) {
+        requireNonNull(pax);
+        checkArgument(isValidPax(pax), MESSAGE_CONSTRAINTS);
+        this.value = pax;
+    }
+
+    /**
+     * Returns true if a given string is a valid pax.
+     */
+    public static boolean isValidPax(String test) {
+        return test.matches(VALIDATION_REGEX);
```

Comment:
I think this returns true if test is "0", which might not be desirable

Reply:
fixed to exclude 0, only allowing positive integers